samples/resources-gallery/web-app/src/jsMain/kotlin/com/icerockdev/web/Main.kt

```diff
@@ -71,6 +71,8 @@ suspend fun main() {
                 Br()
                 Img(Testing.getDrawable().fileUrl)
                 Br()
+                Img(Testing.getVectorDrawable().fileUrl)
```

Comment:
please add same sample in `samples/resources-gallery-compose` to check that vector works with Compose Multiplatform too

Reply:
I've added a sample `svg` to `samples/resources-gallery-compose`

Reply:
I have added a sample to `samples/resources-gallery-compose`. Do you mean to add a new one with fileUrl?

Reply:
sorry, i miss compose change in review. now i check it, all fine